### DISCUSSION TITLE
saf-training/courses/advanced/05

### DISCUSSION THREAD

**Discussion by giscus** [mitre/saf-training#281] [Ideas]
# saf-training/courses/advanced/05

Let's practice creating our own custom resource. Suppose we want to write tests that examine the current state of a local Git repository. We will create a git resource that can ...

https://mitre.github.io/saf-training/courses/advanced/05.html

**Comment by karikarshivani** [comment]
Change the name of the `git` profile to `my_git_profile` to make it easier to differentiate from the `git` resource

**Comment by karikarshivani** [comment]
Change `main` to `testBranch` where applicable

**Comment by karikarshivani** [comment]
The following branches*
